servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/IoExecutor.java

```diff
@@ -60,4 +62,16 @@ default BooleanSupplier shouldOffloadSupplier() {
                 IoThreadFactory.IoThread::currentThreadIsIoThread : // offload if on IO thread
                 Boolean.TRUE::booleanValue; // unconditional
     }
+
+    // FIXME: 0.43 - remove default method
+    @Override
+    default Cancellable execute(Runnable task) throws RejectedExecutionException {
+        throw new UnsupportedOperationException("No existing IoExecutor implementations require this default");
+    }
+
+    // FIXME: 0.43 - remove default method
+    @Override
+    default Cancellable schedule(Runnable task, long delay, TimeUnit unit) throws RejectedExecutionException {
+        throw new UnsupportedOperationException("No existing IoExecutor implementations require this default");
```

Comment:
When we added new methods in 0.41, we usually returned the current class name in the message to let users know which implementation lacks the method. Example: 
https://github.com/apple/servicetalk/blob/100c58bb5098dbcc74ee04778facd68c3f05ebfb/servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java#L177-L180